hyperscale/docker-entrypoint.sh

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/env bash
+set -Eeo pipefail
+# TODO swap to -Eeuo pipefail above (after handling all potentially-unset variables)
+
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
+export POSTGRES_PASSWORD=$(echo $RANDOM | md5sum | head -c 20)
```

Comment:
Maybe add a comment here to say that this is the main difference with the original and why we do this.

Reply:
Fixed